tower-web/src/app/modules/main/component/workflow-status/workflow-status.component.ts

```diff
@@ -25,11 +25,8 @@ export class WorkflowStatusComponent implements OnInit {
 
   private statusesTags: string[];
 
-  statusesRows: string[][];
-
   constructor() {
     this.statusesTags = getAllTaskStatusesProgressStateTags();
-    this.statusesRows = chunk(this.statusesTags, 3)
   }
```

Comment:
Quick question: what's the rationale of this change. I confess I don't know what `chunk` was doing here.

Reply:
`statusesRows` was how you were organising the coloured boxes into two rows of three:
![image](https://user-images.githubusercontent.com/465550/69974325-514a0480-1525-11ea-8a63-2f24429cbe21.png)

To make these fit whatever screen size is available, I refactored the front-end CSS so that these boxes stack into columns automatically. The same 2 rows x 3 columns on large screens, but also 3 x 2 on narrower screens and 6 x 1 on tiny screens. As I was able to do this only using CSS, I was able to delete the back-end code for `statusesRows` as it wasn't needed any more.